test/flows/calculate_statutory_sick_pay_test.rb

```diff
@@ -0,0 +1,598 @@
+                    paid_at_least_8_weeks?: "before_payday",
+                    how_often_pay_employee_pay_patterns?: "weekly",
+                    pay_amount_if_not_sick?: "35000"
+    end
+
+    context "next_node" do
+      should "have a next node of usual_work_days for any valid response" do
+        assert_next_node :usual_work_days?, for_response: "5"
+      end
+    end
+
+    context "validation" do
+      should "be a valid number" do
+        assert_invalid_response "ham"
+      end
+    end
+  end
+
+  context "total_earnings_before_sick_period" do
+    setup do
+      testing_node :total_earnings_before_sick_period?
+      add_responses is_your_employee_getting?: "statutory_adoption_pay",
+                    coronavirus_related?: "no",
+                    employee_tell_within_limit?: "yes",
+                    employee_work_different_days?: "no",
+                    first_sick_day?: "2020-01-01",
+                    last_sick_day?: "2020-01-05",
+                    has_linked_sickness?: "yes",
+                    linked_sickness_start_date?: "2019-12-01",
+                    linked_sickness_end_date?: "2019-12-30",
+                    paid_at_least_8_weeks?: "eight_weeks_less"
+    end
+
+    context "next_node" do
+      should "have a next node of days_covered_by_earnings for any valid response" do
+        assert_next_node :days_covered_by_earnings?, for_response: "700000"
+      end
+    end
+  end
+
+  context "days_covered_by_earnings" do
+    setup do
+      testing_node :days_covered_by_earnings?
+      add_responses is_your_employee_getting?: "statutory_adoption_pay",
+                    coronavirus_related?: "no",
+                    employee_tell_within_limit?: "yes",
+                    employee_work_different_days?: "no",
+                    first_sick_day?: "2020-01-01",
+                    last_sick_day?: "2020-01-05",
+                    has_linked_sickness?: "yes",
+                    linked_sickness_start_date?: "2019-12-01",
+                    linked_sickness_end_date?: "2019-12-30",
+                    paid_at_least_8_weeks?: "eight_weeks_less",
+                    total_earnings_before_sick_period?: "700000"
+    end
+
+    context "next_node" do
+      should "have a next node of usual_work_days for any valid response" do
+        assert_next_node :usual_work_days?, for_response: "5"
+      end
+    end
+  end
+
+  context "usual_work_days" do
+    setup do
+      testing_node :usual_work_days?
+      add_responses is_your_employee_getting?: "statutory_adoption_pay",
+                    coronavirus_related?: "no",
+                    employee_tell_within_limit?: "yes",
+                    employee_work_different_days?: "no",
+                    first_sick_day?: "2020-01-01",
+                    last_sick_day?: "2020-01-05",
+                    has_linked_sickness?: "yes"
+    end
+
+    context "next_node" do
+      context "setup one" do
```

Comment:
What does "setup one" mean? Can it be more descriptive?

Reply:
Sorted